src/applications/check-in/day-of/tests/e2e/extra-validation/validation.failed.cypress.spec.js

```diff
@@ -14,20 +14,33 @@ describe('Check In Experience -- ', () => {
       } = ApiInitializer;
       initializeFeatureToggle.withCurrentFeatures();
       initializeSessionGet.withSuccessfulNewSession();
-      initializeSessionPost.withFailure();
+      initializeSessionPost.withValidationError();
       cy.visitWithUUID();
       ValidateVeteran.validatePageLoaded('Check in at VA');
-      ValidateVeteran.validateVeteran();
-      ValidateVeteran.attemptToGoToNextPage();
     });
     afterEach(() => {
       cy.window().then(window => {
         window.sessionStorage.clear();
       });
     });
-    it('validation failed with failed response from server', () => {
-      Error.validatePageLoaded();
+    it('validation failed with failed response from server. redirect to error page after max validate limit reached', () => {
```

Comment:
Good catch! That case definitely should be accounted for. I may need to modify the session mocks a bit so that we can make the session POST response dynamic, but that shouldn't be too tricky

Reply:
This should be ready now. I updated the mocks to include a dynamic withValidation mock and added additional tests to day-of and pre-check-in that fail once and then validate successfully and advance to the page in the flow.